docs/ensemble_kalman.md

```diff
@@ -0,0 +1,134 @@
+# Ensemble Kalman Smoother
```

Comment:
nice! at some point we can use the `sphinx` package to automatically build these doc pages for all documented functions in the repo and serve it on a website.